components/bl-draggable-list-component/component.json

```diff
@@ -0,0 +1,142 @@
+{
+  "id": "c_f1f837f1781e9409cd5f69da90e17bbf",
+  "name": "Drag and drop list",
+  "description": "This component is a list with floating elements.",
+  "showInToolbox": true,
+  "faIcon": "list",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "name": "itemsArray",
+      "handlerLabel": "Items Array Logic",
+      "handlerDescription": "This is a handler for the logic to determine items array.",
+      "handlerId": "itemsArrayHandler",
+      "type": "text",
+      "dataBinding": true,
+      "label": "Items Array",
+      "defaultValue": "[]"
+    },
+    {
+      "type": "text",
+      "name": "title",
+      "label": "Title",
+      "defaultValue": "List",
+      "handlerId": "titleHandler",
+      "handlerLabel": "Title Logic",
+      "handlerDescription": "This is a handler for the logic to determine title.",
+      "dataBinding": true
+    },
+    {
+      "type": "checkbox",
+      "name": "draggableListVisibility",
+      "label": "Visibility",
```

Comment:
That also may gone when Vlad will added display handler by default for all components